lib/tasks/cap_cutover.rake

```diff
@@ -102,363 +33,13 @@ namespace :cap_cutover do
-          @cap_manual_import_logger.info "Creating brand new Publication '#{row[:article_title]}'  and SourceRecord for #{row[:sunetid]}"
-          pub = Publication.build_new_manual_publication(pub_hash, original_source)
-          pub.save
-        end
-      rescue => e
-        @cap_manual_import_logger.error "Problem with #{row.inspect}"
-        @cap_manual_import_logger.error e.inspect << "\n" << e.backtrace.join("\n")
-      end
-    end
-    @cap_manual_import_logger.info 'Finished import.' + Time.zone.now.to_s
-    @cap_manual_import_logger.info lines.to_s + ' lines of file: ' + args.file_location.to_s + ' were processed.'
-    @cap_manual_import_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
-
-  desc 'Fix authors and pub_hash for manual pubs'
-  task fix_man_pubs_authors: :environment do
-    include ActionView::Helpers::DateHelper
-    start_time = Time.zone.now
-    count = 0
-    @cap_manual_import_logger = Logger.new(Rails.root.join('log', 'fix_man_pubs_authors.log'))
-    @cap_manual_import_logger.info 'Started cap manual pub import process ' + Time.zone.now.to_s
-    header = %("DEPRECATED_PUBLICATION_ID","PUBMED_ID","MANUALLY_ENTERED","PROFILE_ID","CAP_FIRST_NAME","CAP_MIDDLE_NAME","CAP_LAST_NAME","PREFERRED_FIRST_NAME","PREFERRED_MIDDLE_NAME","PREFERRED_LAST_NAME","OFFICIAL_FIRST_NAME","OFFICIAL_MIDDLE_NAME","OFFICIAL_LAST_NAME","SUNETID","UNIVERSITY_ID","EMAIL_ADDRESS","AUTHORSHIP_STATUS","VISIBILITY","FEATURED","PUBLICATION_TITLE","ARTICLE_TITLE","VOLUME","ISSN","ISSUE_NO","PUBLICATION_DATE","PAGE_REF","ABSTRACT","LANG","COUNTRY","AUTHORS","PRIMARY_AUTHOR","AFFILIATION","LAST_MODIFIED_DATE","CAP_IMPORT_TIME","FIRST_PUBLISHED_DATE")
-
-    UserSubmittedSourceRecord.find_each do |usr_src|
-      begin
-        count += 1
-        csv = header + "\n" + usr_src.source_data
-        row = CSV.parse(csv, headers: true, header_converters: :symbol).first
-        pub = usr_src.publication
-        author = pub.authors.first
-        pub_hash = convert_manual_publication_row_to_hash(row, author.id.to_s)
-        pub.pub_hash = pub_hash
-        pub.pubhash_needs_update!
-        pub.save
-        @cap_manual_import_logger.info "Processed #{count}" if (count % 500 == 0)
-      rescue => e
-        @cap_manual_import_logger.error "Problem with UserSubmittedSourceRecord #{usr_src.id}"
-        @cap_manual_import_logger.error "Source Data #{row.inspect}" unless row.nil?
-        @cap_manual_import_logger.error e.inspect << "\n" << e.backtrace.join("\n")
-      end
-    end
-    @cap_manual_import_logger.info 'Finished import.' + Time.zone.now.to_s
-    @cap_manual_import_logger.info lines.to_s + ' lines of file: ' + args.file_location.to_s + ' were processed.'
-    @cap_manual_import_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
```

Comment:
OK, `fix_man_pubs_authors` noted for deprecation in https://github.com/sul-dlss/sul_pub/issues/356#issuecomment-341251365